src/popup.js

```diff
@@ -0,0 +1,90 @@
+import { getData } from './api.js';
+// eslint-disable-next-line import/no-cycle
+import { createCard } from './index.js';
+
+const startPosition = document.querySelector('#body');
+const popup = async () => {
+  const apiData = await getData();
+  const commentBtn = await createCard();
+  let display = '';
+  commentBtn.forEach((e) => {
+    const dataId = e.getAttribute('data-id');
+    e.addEventListener('click', async () => {
+      const response = await fetch(
+        `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/UfLyVv1lq0qsaVGik4HQ/comments?item_id=item${dataId}`,
+      );
+      const commentData = await response.json();
+      display = `<section class="container">
+        <div class="box1">
+        <i class="fa-regular fa-circle-xmark cross"></i> 
+        <img class="char__img" src="${apiData[dataId].imageUrl}"/>
+        <ul class="facts">
+        <li class="fact">Films: ${apiData[dataId].films.length}</li>
+        <li class="fact">Short Films: ${apiData[dataId].shortFilms.length}</li>
+        </ul>
+        <ul class="facts">
+        <li class="fact">Tv Shows: ${apiData[dataId].tvShows.length}</li>
+        <li class="fact">Allies: ${apiData[dataId].allies.length}</li>
+        </ul>
+        </div>
+        <div class="box2">
+        <div>
+          <h3 class="comment__title ">Comments : <span class="counter"></span></h3>
+          <ul id="box" class="comments__box">
+          </ul>
+        </div>
+        <div>
+          <h3 class="comment__title">Add a comment</h3>
+          <form class="form">
+          <input class="username" type="text" required name="username" placeholder="Enter your Name" maxlength="20">
+          <textarea cols="30" rows="5" class="comment" type="text" required name="comment" placeholder="Your Comments" ></textarea>
+          <button type="submit" class="submit-comment">Comment</button>
+          </form>
+        </div>
```

Comment:
- Hey, good job with the design, however, your design layout has not followed the [comment wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). It's meant to be aligned vertically but you've aligned yours horizontally. Please, see the images below to improve the design



*your design*
![Screenshot from 2023-02-10 18-29-34](https://user-images.githubusercontent.com/59397159/218158405-035d340b-931d-491b-b5f7-2d4216b02377.png)

*wireframe design*
![Screenshot from 2023-02-10 18-28-41](https://user-images.githubusercontent.com/59397159/218158410-e7fe3dbb-055a-4274-94dc-cc072c8cf680.png)

--- 

### NOTE

- After you address the import issue, the comment popup won't show. You should kindly fix the issues. :pray:

Reply:
Fixed